assets/less/zenodo-rdm/globals/site.variables

```diff
@@ -3,4 +3,31 @@
  *
  * Invenio App RDM is free software; you can redistribute it and/or modify it
  * under the terms of the MIT License; see LICENSE file for more details.
- */
\ No newline at end of file
+ */
+
+// Colors
+@warningColor: #EB9419;
+@positiveColor: #3A833A;
+@negativeColor: #D33B36;
+
+// Navbar
+@navbarBackgroundImage: linear-gradient(to right,#0047A8 0,#2BBCFF 100%);
+
+
+// Footer colors
+@footerLightColor: #0067c9;
+@footerDarkColor: #0047a8;
+@footerTextLightColor: #c9e5ff;
+
+
+// Font weights
+@normal: 300;
+@semibold: 500;
+@bold: 700;
+
+@headerFontWeight: @semibold;
+
+
+// TODO: REMOVE when https://github.com/inveniosoftware/invenio-app-rdm/pull/1933 is merged
```

Comment:
already merged

Reply:
we need to release this so we can merge it, at least in order to deploy it!